pallet.el

```diff
@@ -237,6 +238,32 @@ use `pallet--package-archives-copy' if USE-COPY is true."
       (message "Pallet: unpacking %s" package-name)
       (pallet--unpack-one package-name))))
 
+;; Command to remove unrequested packages
+(defun pallet-auto-remove-packages ()
+  "Remove packages not explicitly required in Cask file.
+
+This is useful if you synchronize your Cask file across multiple
+machines and rely on pallet to cynchronize your installed
+packages. After deleting a package on one machine and syncing
+your Cask file, run this command on your other machines to
+automatically remove that package on them as well."
```

Comment:
I think we could have both as part of public API (see my further comments above). But for now lets concentrate on just the 'remove surplus packages' functionality, to keep it simple.